README.rst

```diff
@@ -14,18 +14,77 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.0.0 or later.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
+
+::
 
-Please see instructions at http://figure.openmicroscopy.org
+    $ pip install omero-figure
 
+Add figure custom app to your installed web apps:
+
+::
+
+    $ bin/omero config append omero.web.apps '"omero_figure"'
+
+Display a link to 'Figure' at the top of the webclient:
+
+::
+
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "figure"}]' 
+
+Now restart OMERO.web as normal.
+
+**Warning**:
+
+if OMERO.figure is installed with OMERO version prior to **5.2.6**,
```

Comment:
OMERO.figure 2 is designed to be installed into OMERO.web 5.2.6 or newer. This is due to a package re-organization required to distribute the application using Python Package Manager PyPi and Django Framework compatibility. If OMERO.figure 2 is installed into OMERO.web version prior to 5.2.6 urls will change to `https://your-web-server/omero_figure`.